`tsc` error when `--isolatedModules` flag is provided

Using this with typescript works fine so long as you are not using the --isolatedModules flag. The moment that is set to true I get the following three typescript errors:

```
node_modules/react-native-appearance/src/index.tsx:21:3 - error TS1205: Cannot re-export a type when the '--isolatedModules' flag is provided.

21   ColorSchemeName,
     ~~~~~~~~~~~~~~~

node_modules/react-native-appearance/src/index.tsx:22:3 - error TS1205: Cannot re-export a type when the '--isolatedModules' flag is provided.

22   AppearanceListener,
     ~~~~~~~~~~~~~~~~~~

node_modules/react-native-appearance/src/index.tsx:23:3 - error TS1205: Cannot re-export a type when the '--isolatedModules' flag is provided.

23   AppearancePreferences,
```

It would be great if this were updated to include support for the isolatedModules flag! Thank you and please let me know if I can provide any further information!

--- a/src/index.native.tsx
+++ b/src/index.native.tsx
@@ -12,13 +12,6 @@ if (!Appearance) {
   useColorScheme = polyfill.useColorScheme;
 }
 
-import { ColorSchemeName, AppearanceListener, AppearancePreferences } from './Appearance.types';
+export * from './Appearance.types';
 
-export {
-  Appearance,
-  AppearanceProvider,
-  useColorScheme,
-  ColorSchemeName,
-  AppearanceListener,
-  AppearancePreferences,
-};
+export { Appearance, AppearanceProvider, useColorScheme };

--- a/src/index.web.tsx
+++ b/src/index.web.tsx
@@ -1,5 +1,3 @@
-import { AppearanceListener, AppearancePreferences, ColorSchemeName } from './Appearance.types';
+export * from './Appearance.types';
 
 export * from './polyfill';
-
-export { ColorSchemeName, AppearanceListener, AppearancePreferences };

